app/controllers/zendesk/tickets_controller.rb

```diff
@@ -13,10 +14,16 @@ def show
         zendesk_instance_domain: EitcZendeskInstance::DOMAIN
       )
       @document_groups = DocumentPresenter.grouped_documents(@intakes)
+
+      @drop_offs = IntakeSiteDropOff.where(zendesk_ticket_id: zendesk_ticket_id)
```

Comment:
we considered modifying the `DocumentPresenter` to handle both types of records but it seemed like overkill since preparers are the ones uploading documents for drop off, all the extra info is probably less helpful. let us know if the code is too repetitive as a result!

Reply:
I think that's fine, we can add if needed later. Good call.